### ISSUE TITLE
Streamline registration for patrons with qualifying print disabilities

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#10457]
### Problem / Opportunity

For patrons who have qualified print disabilities, getting access to a `special access` account is challenging.

#### How it works [or doesn't] today

Currently, Internet Archive patrons _hypothetically_ have 2 primary ways of learning about qualified access for patrons with print disabilities:

1. By navigating to an Internet Archive book with limited or restricted availability

e.g. borrowable/limited https://archive.org/details/10000ideasforter00lamm and knowing/trying to clicking the dropdown:
<kbd>
<img width="649" alt="Image" src="https://github.com/user-attachments/assets/a05ab2bf-0244-4c13-a7bb-588c15881902" />
</kbd>
e.g. restricted https://archive.org/details/1000sneakersguid0000lema ("print disabled" is not a link and the FAQs don't explain print disabled program)

<kbd>
<img width="657" alt="Image" src="https://github.com/user-attachments/assets/d60a5565-0b48-401d-adf5-eb7e5b82ebf7" />
</kbd>

2. By landing on the **help** pages, either by a search engine query or (and I'm not sure how a patron would know to do this) by clicking the "next" page of the help carousels until they see the "print disabled resources" card:

<kbd>
<img width="874" alt="Image" src="https://github.com/user-attachments/assets/b47f6e21-41ca-4a8e-afd4-88680c1bf992" />
<img width="900" alt="Image" src="https://github.com/user-attachments/assets/f8e8ccf2-002f-4647-bf53-622264dc43c1" />
</kbd>

From the help page https://help.archive.org/help/print-disability-access-general-information:

<kbd>
<img width="716" alt="Image" src="https://github.com/user-attachments/assets/8400b302-a206-4ce0-be45-0dca5963c969" />
</kbd>

The patron would have to click the "special disability link", which is incorrectly formatted and goes to `https://archive.org/details/printdisabled&tab=about` instead of `https://archive.org/details/printdisabled?tab=about` -- however this currently shows the patron this error:

<kbd>
<img width="1001" alt="Image" src="https://github.com/user-attachments/assets/b7e2d88f-c8b2-4661-9056-f3d742fbe575" />
</kbd>

From the error page, the patron can't even navigate to the `about` tab (even if they knew to) because the URL structure will remain broken.

The patron would somehow have to know to fix the URL in order to arrive at: https://archive.org/details/printdisabled?tab=about:

<kbd>
<img width="183" alt="Image" src="https://github.com/user-attachments/assets/e1b3db58-cd2b-4525-a7f7-e16d21c832e9" />

<img width="755" alt="Image" src="https://github.com/user-attachments/assets/5f1b0785-c0d9-4a4c-a755-d4a41d59b78f" />
</kbd>

Once at the Print Disabled page, one may navigate to the link [fill in this form to contact the Vermont Mutual Aid Society](https://docs.google.com/forms/d/e/1FAIpQLScSBbT17HSQywTm-fQawOK7G4dN-QPbDWNstdfvysoKTXCjKA/viewform) to begin the registration process.

#### Justify Impact

No only is this process being made complicated for the class of patron who require accessibility affordances the most, but also many qualifying patrons are likely not discovering or failing to register for our services. 

#### What does Success look like?

Success looks like a link and a single click from the Open Library and Archive.org registration page to verify print disability status or apply for special access.

The current [application](https://docs.google.com/forms/d/e/1FAIpQLSfClhUkTvF5z29hhxoi5UgaB1_50lr8F7HV5Cq0ABdztX9CkA/viewform) form has about 1.5k submissions.

EDIT: @jessamynwest writes below:
> [The forms] are usually archived when they hit 1000 submissions because the sheets gets unwieldy. The program gets about 1000 submissions per month give or take currently

Open Library's goal for 2025 is to facilitate a 10% increase, which is +1.2k more submissions (+100 per month)

##### Metrics

- How many clicks to submission form
- How many submissions? (from OL, IA)
- How many submissions accepted v. rejected

### Proposal

Integrate the qualification and submission process into the Open Library registration form:

<kbd>
<img width="393" alt="Image" src="https://github.com/user-attachments/assets/3f55c4f2-9828-4685-902d-3814a1565544" />
</kbd>

We should link to a"Who qualifies" section similar to https://www.bookshare.org/help-and-learning-articles/who-qualifies-for-bookshare

On the registration complete / activate screen, we can include a form that asks the patron to select an organization from a dropdown, etc.

This is what Bookshare registration, a similar service, looks like:
https://www.bookshare.org/sign-up

### Related files

Refer to [this map of common Endpoints](https://github.com/internetarchive/openlibrary/wiki/Endpoints):
* [/account/login](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/upstream/account.py#L399-L476)
* [login.html](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/login.html)
* [form validations](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/upstream/forms.py)
* [Related PR](https://github.com/internetarchive/openlibrary/pull/9579)

### Requirements Checklist
Checklist of requirements that need to be satisfied in order for this issue to be closed:
* [ ] Add checkbox to registration form for print disability access
* [ ] Work with @mekarpeles to create a "who qualifies" page that registration may link to
* [ ] Update the registration success / activate page to include a registration form to apply for special access

### Stakeholders

* @bfalling -- archive.org team lead
* Andrea Mills + @jessamynwest -- leads on special access for those with qualifying print disabilities
* @brewsterkahle -- final decision maker

<hr>

### Instructions for Contributors

* **Before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) or pushing up changes to a PR, please first [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date, as the pre-commit bot may add commits to your PRs upstream.

**Comment by jessamynwest** [comment]
> The current form has about 1.5k submissions

Just a note, that form is just the top sheet. They are usually archived when they hit 1000 submissions because the sheets gets unwieldy. The program gets about 1000 submissions per month give or take currently. We've processed 43,000 responses since December 2019. My understanding is that there may be internal reasons why there is some friction built into this process but I will defer to folks who work at the Archive.

**Comment by mekarpeles** [comment]
Thank you @jessamynwest! Super useful. If we're getting ~1k / mo, this is well aligned with our original goal, which is to facilitate +1k submissions (from Open Library) in 2025, which would constitute ~10% increase (+100 / mo on top of our existing numbers). The archival process and lack of indication in the form of the source of the submission (e.g. Open Library, Archive.org, other) may make it a bit tricky to know how many forms were submitted that originated at Open Library, but we can always decide to just count the number of times a patron registers with intent (even if they don't submit the form) as a baseline, find a way to automatically pass along / prefill the origin (and ideally their archive.org account) into the form, and we could also consider future steps to organize the submissions in a way that doesn't require manually maintaining and archiving google sheets.

**Comment by jessamynwest** [comment]
> ideally their archive.org account

In the past I had tried to make this form behind a paywall to capture the archive.org account (my biggest time-consumer is people who apply who either don't have an account or have an account at the Archive which is different from the one that autofills on the Google form) and that never rose to the level of being actively worked on.

> we could also consider future steps to organize the submissions in a way that doesn't require 
> manually maintaining and archiving google sheets

Please speak with Andrea/Brenton about the bright line which needs to be between the work that VTMAS does and the work of the Archive. I am happy to stay updated through Andrea because, as a volunteer, I do not have the time to also be an active part of this project.

**Comment by mekarpeles** [comment]
Thank you as always @jessamynwest, I appreciate how much time and energy you've put into serving audiences with print disabilities and am eager to help support in ways that are useful.

If Open Library registration includes a checkbox to submit a special access request for a qualifying print disability, we could potentially use a google form "pre-fill" link to pass their archive.org ID into the google form that gets open.

In fact, already on this page: https://archive.org/details/printdisabled?tab=about

We **should** know if they are logged in and, when they are, could also automatically pass their archive.org username to this form.

See: https://www.youtube.com/watch?v=z4MOgd6JS_0

I can coordinate with @bfalling to see if we may be able to customize the link we use and fill it in with their archive.org username **when logged in**.
